packages/paste-website/src/components/site-wrapper/SiteBody.tsx

```diff
@@ -95,24 +74,28 @@ const PasteThemeAlert: React.FC<{isPasteTheme: boolean}> = ({isPasteTheme}) => {
   );
 };
 
+export const SiteBodyContext = React.createContext({isPasteTheme: false});
+
 export const SiteBody: React.FC = ({children}) => {
   const {theme: activeTheme} = useActiveSiteTheme();
   const isPasteTheme = activeTheme === 'default';
   return (
-    <>
-      <PsaAlert />
-      <PasteThemeAlert isPasteTheme={isPasteTheme} />
-      <StyledSiteBody isPasteTheme={isPasteTheme}>
-        <Sidebar />
-        <Box height="100vh" overflow="auto">
-          <SiteHeader />
+    <SiteBodyContext.Provider value={{isPasteTheme}}>
+      <Box position="sticky" top="0" zIndex="zIndex20">
+        <PsaAlert />
+        <PasteThemeAlert isPasteTheme={isPasteTheme} />
```

Comment:
Good call on the context provider! I think we aughta remove these `isPasteTheme` props being drilled down everywhere and just rely on the context in all these components.

Reply:
Yeah, that's fair. I can remove and just rely on the context